editor/script_editor_debugger.cpp

```diff
@@ -642,7 +643,7 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 							v /= 1024.0;
 						}
 						tt += " bytes";
-						vs = rtos(v) + " " + unit;
+						vs = String::num(v, 2) + " " + unit;
 					} break;
 					case Performance::MONITOR_TYPE_TIME: {
 						tt += " seconds";
```

Comment:
ugh, i forgot to TTR this.

Reply:
I thought about that also. but I doubted if it's necessary.